### PR TITLE
Virology symptom tweaks and a new symptom

### PR DESCRIPTION
**Pull request by ghost**
- Messages have been added to many symptoms that lack them
- Some messages that should be more threatening have had their spans changed (i.e. notice -> warning)
- Spontaneous Combustion now does small amounts of fire damage depending on the stage and causes the host to scream if it's at max stage
- The messages for the hallucination symptom have been changed
- **New symptom: Weakness.** This symptom will inflicts small amounts of stamina damage at lower stages and moderate amounts at higher stages, plus brief fainting spells if the host has too much stamina damage (this is to let them sleep some of it off, as sleeping increases stamina healing dramatically). Get it from unstable mutagen.

### CODE REVIEW

**Comment by Xhuis** [comment]
also what the fuck is with this tgstation.dme diff

**Comment by PKPenguin321** [comment]
what do you need to use to get Weakness? virus food, mutagen, or plasma?

**Comment by Xhuis** [comment]
uhh
how do i figure that out

File: code/datums/diseases/advance/symptoms/weakness.dm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+*/
+
+/datum/symptom/weakness
+
+	name = "Weakness"
+	stealth = -3
+	resistance = -2
+	stage_speed = -2
+	transmittable = -4
+	level = 3
```
Comment (PKPenguin321) [inline code/datums/diseases/advance/symptoms/weakness.dm]: pretty sure this determines what chem is used to roll for the symptom
from the wiki: "You can obtain symptom levels 1 and 2 with Virus Food, which you start with a dispenser full of. For symptom levels 3 and 4 you will need to acquire Unstable Mutagen. Levels 5 and 6 requires liquid plasma."
3 or 4 is probably good for this symptom

Comment (Xhuis) [inline code/datums/diseases/advance/symptoms/weakness.dm]: ahh okey

**Comment by Xhuis** [comment]
I modified the values on the Weakness symptom. It's now only slightly noticeable and moderately decreases other stuff about the disease.

This is merge-ready.

**Comment by Lumipharon** [comment]
Wait, does this make combustion set you on fire AND do arbitrary extra fire damage?
It's already pretty damn good you know. Oh well.

**Pull request merged**